test/unit/windowHandlers.test.js

```diff
@@ -61,6 +85,333 @@ describe("Window handlers test", function () {
         });
     });
 
+    describe("Init tests for recipes", function () {
```

Comment:
why are these tests added to windowhandlers test file? These have nothing to do with window handlers..

Reply:
Moved them to a separate file